src/GithubPlugin/Helpers/TimeSpanHelper.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 using System.Text.Json.Nodes;
 using DevHome.Logging;
+using Jeffijoe.MessageFormat;

```

Comment:
My only concern is about this: if "Jeffijoe" is a library that is okay to use. At all, LGTM.